tests/test_clpir.py

```diff
@@ -0,0 +1,280 @@
+import dateutil.tz
+import pickle
+import unittest
+
+from clp_ffi_py.CLPIR import LogEvent, Metadata
+from datetime import tzinfo
+from typing import Optional
+
+
+class TestCaseMetadata(unittest.TestCase):
+    """
+    Class for testing clp_ffi_py.CLPIRDecoder.Metadata.
+    """
+
+    def check_metadata(
```

Comment:
Ideally make these helper methods private (prefix with `__`) and move them under the tests according to our ordering rules (public before private, etc.).